File: app/dbmodels/DeviceProcessor.php
<?php


$table['model'] = 'DeviceProcessor';

include "_commondevices.php";

$table['fields']['busID']     = DBModels::type('string');
$table['fields']['frequency'] = DBModels::type('integer');
$table['fields']['nbcores']   = DBModels::type('integer');
$table['fields']['nbthreads'] = DBModels::type('integer');

File: fusionglpi/database/device_processor.py
from fusionglpi import db


class DeviceProcessor(db.Model):
    id = db.Column(db.Integer, primary_key=True)
    name = db.Column(db.String(255, None, True))
    comment = db.Column(db.Text)
    busID = db.Column(db.String(255, None, True))
    frequency = db.Column(db.Integer, nullable=False, server_default="0")
    nbcores = db.Column(db.Integer, nullable=False, server_default="0")
    nbthreads = db.Column(db.Integer, nullable=False, server_default="0")
